bin/logstash

```diff
@@ -39,13 +39,19 @@ else
   SOURCEPATH="$0"
 fi
 
-. "$(cd `dirname $SOURCEPATH`/..; pwd)/bin/logstash.lib.sh"
+. "$(cd `dirname ${SOURCEPATH}`/..; pwd)/bin/logstash.lib.sh"
 setup
 
-if [ "$1" = "-V" ] || [ "$1" = "--version" ];
-then
-  LOGSTASH_VERSION_FILE="${LOGSTASH_HOME}/logstash-core/lib/logstash/version.rb"
-  LOGSTASH_VERSION="$(sed -ne 's/^LOGSTASH_VERSION = "\([^*]*\)"$/\1/p' $LOGSTASH_VERSION_FILE)"
+if [ "$1" = "-V" ] || [ "$1" = "--version" ]; then
+  LOGSTASH_VERSION_FILE1="${LOGSTASH_HOME}/logstash-core/versions-gem-copy.yml"
```

Comment:
Can you add a comment here to why we check 2 files?

Also, nitpick...but can you name the file  copied file versions.yml , you already have a comment in copy and I think it makes it more obvious they are the same file, if they are named he same.

Reply:
Re: comment - sure.

Re: nit-pick, well if you look at the code in `logstash-core/lib/logstash/version.rb` and `logstash-core/lib/logstash-core/version.rb`, line 8 - I wanted it to crystal clear that the file is *not* the root `versions.yml` because trying to work out which folder `../../` is pointing at is hard (for me at least). Oh and people don't read comments!

Reply:
>Oh and people don't read comments!

Can't argue with that :)